slasher/service/data_update.go

```diff
@@ -143,6 +140,29 @@ func (s *Service) detectSlashings(idxAtt *ethpb.IndexedAttestation) error {
 	return nil
 }
 
+func (s *Service) getCommittees(at *ethpb.Attestation) (*ethpb.BeaconCommittees, error) {
+	epoch := at.Data.Target.Epoch
+	committees, err := committeesCache.Get(s.context, epoch)
```

Comment:
Why are we using the service context? Should the input context be separated from that?

```suggestion
	committees, err := committeesCache.Get(ctx, epoch)
```

Reply:
+1 nice catch, we should pass in ctx as the first argument to this getCommittees function